test/ProgressOnderwijsUtilsTests/CollectionBehavior.cs

```diff
@@ -0,0 +1,3 @@
+﻿using Xunit;
+
+[assembly: CollectionBehavior(DisableTestParallelization = true)]
```

Comment:
fine of course, but out of curiosity: what went wrong with concurrent test execution?

Reply:
The TVar_ creation stuff.